### ISSUE TITLE
Video pauses when moving to a different Stage Manager stage

### ISSUE THREAD

**Issue by sizemo** [fer0n/Unwatched#21]
Hi there,

First off, thank you so much for making this app! It’s incredible and it’s changed how I watch YouTube. 

I’m primarily using Unwatched on my iPad where I have Stage Manager enabled. Sometimes, when I click on a different app, it takes me to a different stage and Unwatched stops playing. The behavior I’d expect would be for the video to pop out and follow me to the next stage instead of just pausing.

I understand it’s related to the video player, because I enabled the force fullscreen native viewer  option and the video behaved as expected when I had that on, but I didn’t love having to have it fullscreen all the time with that feature. Is it possible to enable a native player view without making it fullscreen?

Thank you!

**Comment by fer0n** [comment]
Are you able to enter fullscreen manually by tapping the fullscreen button on the bottom right inside the video?

**Comment by sizemo** [comment]
Yes I am, but for the video I was watching, it was still using the YouTube player when I tapped that button to go fullscreen rather than the native Apple one.

**Comment by fer0n** [comment]
This one? 
![image](https://github.com/user-attachments/assets/1c51f4b2-463d-4acc-821a-5d8139a42cce)

**Comment by fer0n** [comment]
I have since added a dedicated Pip button, which should make enabling it much more approachable. I sadly haven't found a way to automatically enter Pip when the App enters the background, which would be helpful for other areas as well.
